vmdb/spec/controllers/catalog_controller_spec.rb

```diff
@@ -135,7 +130,7 @@
     it "Orchestration Template is copied" do
       controller.instance_variable_set(:@sb, {})
       controller.instance_variable_set(:@_params, :button => "save")
-      controller.instance_variable_set(:@_response, ActionController::TestResponse.new)
+      controller.should_receive(:render)
       ot = FactoryGirl.create(:orchestration_template)
       controller.x_node = "xx-ot_othot-#{ot.id}"
```

Comment:
Actually, I'd have to do something like:

controller.instance_variable_set(:@sb, :active_tree => "ot_tree", :trees => {"ot_tree" => {:active_node => "xx-othot_ot-#{ot.id}"}})

Doing controller.x_node= seems easier to me.